README.md

```diff
@@ -75,7 +75,7 @@ jfrogClient.xray().summary().component({
 #### Retrieving Xray Build Details
 
 ```javascript
-jfrogClient.xray().details().build('Build Name', '1')
+jfrogClient.xray().details().build('Build Name', '1', 'Project Key')
```

Comment:
Since the *project_key* argument is optional. let's add a comment here noting that.